src/components/Search/Search.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import styles from './Search.css';
+
+const Search = ({ artistText, artists, handleChange, handleSubmit }) => {
+  const artistNodes = artists.map(artist => (
```

Comment:
You should try out those HOC withList things, I think you'd really like it and could grasp it well